backend/dataall/modules/datasets/cdk/dataset_stack.py

```diff
@@ -242,7 +242,8 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
                     sid="ListDatasetBucket",
                     actions=[
                         "s3:ListBucket",
-                        "s3:GetBucketLocation"
+                        "s3:GetBucketLocation",
+                        "s3:GetBucketAcl"
```

Comment:
Is this extra `s3:GetBucketAcl` permissions needed because Dataset Role now handling file uploads to S3 bucket using presigned url?